modules/RaptMediaNg/RaptMediaNg.json

```diff
@@ -0,0 +1,34 @@
+{
+	"mw.RaptMediaNg": {
+		"scripts": [
+			"resources/engine.min.js",
```

Comment:
if this exists on the previous rapt implementation - we can use the same file and not duplicate it

Reply:
Hi @eitanavgil,

Thank you for taking the time to review.
Technically speaking, we could use resources/engine.min.js, resources/RaptMediaNgPlugin.css and resources/mw.RaptMediaNgDurationLabel.js as they exist in the original `RaptMedia` plugin [modules/RaptMedia/resources]. The reason why I duplicated these [and renamed them accordingly] is to make the new plugin as independent/self contained as possible so that, if in the future, changes are made to the original, they will not affect the new one and, by the same token, should the new plugin necessitate changes in these files for whatever reason, they will not affect the original. Does that make sense?